docs/reference/upgrade.asciidoc

```diff
@@ -68,10 +71,6 @@ When upgrading to a new version of {es}, you need to upgrade each
 of the products in your Elastic Stack. For more information, see the
 {stack-ref}/upgrading-elastic-stack.html[Elastic Stack Installation and Upgrade Guide].
 
-To upgrade directly to {version} from 6.7 or earlier, you must shut down the
```

Comment:
I think we should leave this here. It's more general than the text in the `WARNING` block above (6.7 _or earlier_) and I don't think it needs to be part of a `WARNING` at all.

Reply:
If it is ok for you, I've kept it but moved up closed to the other upgrade paths.